@xen-orchestra/web-core/lib/locales/fr.json

```
@@ -36,38 +30,59 @@
   "core.sidebar.lock": "Verrouiller la barre latérale",
   "core.sidebar.open": "Ouvrir la barre latérale",
   "core.sidebar.unlock": "Déverrouiller la barre latérale",
-
   "core.sort.ascending": "Trier par ordre croissant",
   "core.sort.descending": "Trier par ordre décroissant",
-
   "dark-mode.enable": "Activer le mode sombre",
   "dark-mode.disable": "Désactiver le mode sombre",
   "dark-mode.auto": "Mode sombre automatique",
-
   "access-forum": "Accès au forum",
   "connected": "Connecté",
+  "copy-info-json": "Copier les informations en JSON",
   "dashboard": "Tableau de bord",
+  "default-locking-mode": "Mode de verrouillage par defaut",
+  "dns": "DNS",
+  "delete": "Supprimer",
+  "description": "Description",
+  "device": "Appareil",
+  "disabled": "Désactivé",
   "disconnected": "Déconnecté",
-  "disconnected-from-physical-device": "Déconnecté de l'appareil physique",
+  "disconnected-from-physical-device": "Déconnécté de l'appareil physique",
```

Please roll back the line:
```suggestion
  "disconnected-from-physical-device": "Déconnecté de l'appareil physique",
```